test/lit.site.cfg.in

```diff
@@ -34,6 +34,7 @@ config.android_ndk_path = "@SWIFT_ANDROID_NDK_PATH@"
 config.android_ndk_gcc_version = "@SWIFT_ANDROID_NDK_GCC_VERSION@"
 
 config.coverage_mode = "@SWIFT_ANALYZE_CODE_COVERAGE@"
+config.lldb_build_root = "@LLDB_BUILD_DIR@"
```

Comment:
Ah, we need this line in `validation-test/lit.site.cfg.in` too. Otherwise,
```
  File "/home/rintaro/Documents/swift-oss/swift/test/lit.cfg", line 1073, in <module>
    if config.lldb_build_root != "":
AttributeError: TestingConfig instance has no attribute 'lldb_build_root'
```